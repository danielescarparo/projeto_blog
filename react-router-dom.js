import React from "react";

export const Link = () => <a> link </a>;
export const NavLink = ({children}) => <div> {children} </div>;