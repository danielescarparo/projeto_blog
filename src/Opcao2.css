.group-rec{
    display: flex;
    justify-content: space-around;
    flex-wrap: wrap;
}

.rectangle{
    /* justify-content: center;
    align-items: center; */
    width: 100px;
    height: 100px;
    margin: 50px;
    background: #10718c;
}