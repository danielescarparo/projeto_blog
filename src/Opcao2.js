import React, { Component } from 'react';
import {NavLink as Link} from 'react-router-dom'

import './Opcao2.css';

class Opcao2 extends Component {
  render() {
    return (
      <div className="group-rec">
        <Link className="rectangle" to="/opcao2/1">1</Link>
        <Link className="rectangle" to="/opcao2/2">2</Link>
        <Link className="rectangle" to="/opcao2/3">3</Link>
        <Link className="rectangle" to="/opcao2/4">4</Link>
        <Link className="rectangle" to="/opcao2/4">5</Link>
        <Link className="rectangle" to="/opcao2/1">1</Link>
        <Link className="rectangle" to="/opcao2/2">2</Link>
        <Link className="rectangle" to="/opcao2/3">3</Link>
        <Link className="rectangle" to="/opcao2/4">4</Link>
        <Link className="rectangle" to="/opcao2/4">5</Link>
        <Link className="rectangle" to="/opcao2/1">1</Link>
        <Link className="rectangle" to="/opcao2/2">2</Link>
        <Link className="rectangle" to="/opcao2/3">3</Link>
        <Link className="rectangle" to="/opcao2/4">4</Link>
        <Link className="rectangle" to="/opcao2/4">5</Link>
        <Link className="rectangle" to="/opcao2/1">1</Link>
        <Link className="rectangle" to="/opcao2/2">2</Link>
        <Link className="rectangle" to="/opcao2/3">3</Link>
        <Link className="rectangle" to="/opcao2/4">4</Link>
        <Link className="rectangle" to="/opcao2/4">5</Link>
      </div>  
    );
  }
}

export default Opcao2;
