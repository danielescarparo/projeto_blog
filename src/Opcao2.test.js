import React from 'react';
import ReactDOM from 'react-dom';
import Opcao2 from './Opcao2';
import Adapter from 'enzyme-adapter-react-16';
import {BrowserRouter, Switch, Router} from 'react-router-dom' 
import { shallow, configure } from 'enzyme'; 
configure({adapter: new Adapter()});

it('renders without crashing', () => {
  const wrapper = shallow(<BrowserRouter><Opcao2/></BrowserRouter>);
  expect(wrapper.html()).toMatchSnapshot();
});

