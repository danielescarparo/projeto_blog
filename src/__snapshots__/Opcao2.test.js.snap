// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `"<div class=\\"group-rec\\"><a class=\\"rectangle\\" href=\\"/opcao2/1\\">1</a><a class=\\"rectangle\\" href=\\"/opcao2/2\\">2</a><a class=\\"rectangle\\" href=\\"/opcao2/3\\">3</a><a class=\\"rectangle\\" href=\\"/opcao2/4\\">4</a><a class=\\"rectangle\\" href=\\"/opcao2/4\\">5</a><a class=\\"rectangle\\" href=\\"/opcao2/1\\">1</a><a class=\\"rectangle\\" href=\\"/opcao2/2\\">2</a><a class=\\"rectangle\\" href=\\"/opcao2/3\\">3</a><a class=\\"rectangle\\" href=\\"/opcao2/4\\">4</a><a class=\\"rectangle\\" href=\\"/opcao2/4\\">5</a><a class=\\"rectangle\\" href=\\"/opcao2/1\\">1</a><a class=\\"rectangle\\" href=\\"/opcao2/2\\">2</a><a class=\\"rectangle\\" href=\\"/opcao2/3\\">3</a><a class=\\"rectangle\\" href=\\"/opcao2/4\\">4</a><a class=\\"rectangle\\" href=\\"/opcao2/4\\">5</a><a class=\\"rectangle\\" href=\\"/opcao2/1\\">1</a><a class=\\"rectangle\\" href=\\"/opcao2/2\\">2</a><a class=\\"rectangle\\" href=\\"/opcao2/3\\">3</a><a class=\\"rectangle\\" href=\\"/opcao2/4\\">4</a><a class=\\"rectangle\\" href=\\"/opcao2/4\\">5</a></div>"`;
